tests/test_list.py

```diff
@@ -27,6 +27,14 @@ def test_cli(pipx_temp_env, monkeypatch, capsys):
     assert "nothing has been installed with pipx" in captured.err
 
 
+def test_cli_global(pipx_temp_env, monkeypatch, capsys):
+    if sys.platform.startswith("win"):
+        pytest.skip("This behavior is undefined on Windows")
+    assert not run_pipx_cli(["--global", "list"])
+    captured = capsys.readouterr()
+    assert "nothing has been installed with pipx" in captured.err
```

Comment:
Like this this test seems not very useful. I think installing a package without `--global` before running `list` with `--global` would make this much more useful.

Reply:
Good idea! Added install without `--global`.